### PR TITLE
[6] Product Page Using Html

### PR DESCRIPTION
**Pull request by ElJentle**
## Task: `Product Page Using Html`
- `create html task branch, add index.html file`

Web page 1
Web page 2
Web page 3

### CODE REVIEW

**Comment by ElJentle** [comment]
> Well done @ElJentle
> 
> I think you should create an `images` folder inside your `htmlTask` folder and place all the images you are using there.
> Also, the names of your images shouldn't have space E.g `image pro.png` can be properly written as `image_pro.png`. Same goes for others too



I have updated the PR based on the comment above.

Many thanks.

File: submissions/emmanuelEsuong/htmlTask/index.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/emmanuelEsuong/htmlTask/index.html:1]: You need to remove all the styles in this file. From font to the images.

File: submissions/emmanuelEsuong/htmlTask/index.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+        </pre>
+    
+        <pre>
+        <p></pre>
+            Shop at the Best Prices and Get Your Order Delivered at Your Door-Step in a Jiffi
+        </pre></p>
+        
+
+        <form action="/form/submit" method="GET"><input type="text" name="text" class="search" placeholder="Search here!"><input type="submit" name="submit" class="submit" value="Search"</form>
```
Comment (ToluAkin) [inline submissions/emmanuelEsuong/htmlTask/index.html:53]: You need to remove the action and method element

Comment (ElJentle) [reply submissions/emmanuelEsuong/htmlTask/index.html]: Element removed.

**Pull request merged**